From 3a10c7aac8096301256df24feeac7848d4ab0dc0 Mon Sep 17 00:00:00 2001
From: Alexandre Ghiti <alexghiti@rivosinc.com>
Date: Wed, 13 Nov 2024 14:10:39 +0000
Subject: [PATCH] Do not run stack_grow_into_huge

This test is flaky and is intended to test a powerpc feature

Signed-off-by: Alexandre Ghiti <alexghiti@rivosinc.com>
---
 tests/run_tests.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/run_tests.py b/tests/run_tests.py
index 018264d..a2a0b6e 100755
--- a/tests/run_tests.py
+++ b/tests/run_tests.py
@@ -572,7 +572,7 @@ def functional_tests():
     do_test("truncate_above_4GB")
     do_test("brk_near_huge")
     do_test("task-size-overrun")
-    do_test_with_rlimit(resource.RLIMIT_STACK, -1, "stack_grow_into_huge")
+    #do_test_with_rlimit(resource.RLIMIT_STACK, -1, "stack_grow_into_huge")
     do_test("corrupt-by-cow-opt")
     do_test("noresv-preserve-resv-page")
     do_test("noresv-regarded-as-resv")
-- 
2.34.1

